src/todo-list.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Created by idoi on 25/03/2018.
+ */
```

Comment:
General:
Consider using 'document.querySelector' and 'document.querySelectorAll' instead of 'getElementById'.